{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -29,6 +29,14 @@ requirements: test_environment
 data: requirements
 	$(PYTHON_INTERPRETER) src/data/make_dataset.py
 
+## Train Model
```

Comment:
These should only be included if the option is selected. Please use a template conditional.